ert_shared/models/ensemble_smoother.py

```diff
@@ -32,11 +32,11 @@ def runSimulations(self, arguments):
         self.setPhaseName("Pre processing...", indeterminate=True)
         self.ert().getEnkfSimulationRunner().createRunPath(prior_context)
```

Comment:
Is there some "hidden" magic behding the getter of the Simulation Runner which makes it not ok to assign it to a variable in the beginning of the functions instead of getting it multiple times in the same function? Not a big deal - just noticing it 😄

Reply:
I am not sure. Could be ok, but I am not touching it for this PR ;)